linera-core/src/unit_tests/client_tests.rs

```diff
@@ -509,7 +509,7 @@ async fn test_memory_transfer_then_open_chain() -> Result<(), anyhow::Error> {
 #[test(tokio::test)]
 async fn test_rocksdb_transfer_then_open_chain() -> Result<(), anyhow::Error> {
```

Comment:
So the problem is that to be consistent, the following should also be renamed about everywhere:
`rocksdb` -> `rocks_db`
`ROCKSDB` -> `ROCKS_DB`

This could be an issue for the feature named "rocksdb" that is visible in `crates.io`.

Reply:
Ok, I renamed it accordingly. Now, we just have the "rocksdb" feature and the rocksdb package.